cuML/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ else()
 	message(STATUS "Manually setting BLAS to ${BLAS_LIBRARIES}")
 endif()
 
-find_package(CUDA 9.0 REQUIRED)
+find_package(CUDA 10.0 REQUIRED)
```

Comment:
This is a HUGE change that would require serious discussion in the whole team, why make CUDA 10 required?

Reply:
My mistake it was for development, I have been testing with the magma-cuda10 package. This change is not necessary. It also leads me to another question about the need to condition the pip install on the available cuda version for the cuml package, as there are two package magma-cuda10 and magma-cuda92